src/main/webapp/index.html

```diff
@@ -48,6 +49,13 @@ <h1>Census Viz
             in 0, you'll get information from the larger Decennial Survey instead.
           </p>
         </div>
+        <div class="tooltip">
+          <i class="material-icons" id="location-off-icon" onclick="changeLocationSettings(this)">location_off</i>
+          <i class="material-icons" id="location-on-icon" onclick="changeLocationSettings(this)">location_on</i>
+          <p class="tooltiptext">
+            Click to change location settings
```

Comment:
I love the way the icon changes really seamlessly when you click on it!
Could you update the tooltip text a bit to be more clear about what the settings are? Like "Click to toggle location data collection and location-based suggestions" or something like that.
(Also, we should bring the language of this tooltip and the feedback one in line, so either both are "Click" or both are "Click here", and both end in a period or both end without punctuation. But either of those options works just fine to me.)

Reply:
Also, maybe this is tricky, but is there any way we could make it clear whether, if I see the icon with a slash through it, that means location is currently off, or it's currently on and I could click to turn it off? I just realized that I actually have no idea which is which.
Maybe the easiest way to do this would be to have your JS changeLocationSettings function also change this particular tooltiptext to be "Location data off. Click to ..." or "Location data on. Click to ..." as appropriate.

Reply:
This is a great suggestion! Will look into this